src/test/java/org/folio/rest/RestTestSuite.java

```diff
@@ -0,0 +1,26 @@
+package org.folio.rest;
+
+import org.folio.rest.impl.StorageHelperTest;
+import org.folio.rest.impl.TenantRefAPITest;
+import org.folio.rest.support.CollectionUtilTest;
+import org.folio.rest.support.CompletableFutureUtilTest;
+import org.folio.rest.support.CqlQueryTest;
+import org.folio.rest.unit.ItemDamagedStatusAPIUnitTest;
+import org.junit.runner.RunWith;
+import org.junit.runners.Suite;
+
+@RunWith(Suite.class)
+@Suite.SuiteClasses({
+  CollectionUtilTest.class,
+  CompletableFutureUtilTest.class,
+  CqlQueryTest.class,
+  ItemDamagedStatusAPIUnitTest.class,
+  StorageHelperTest.class,
+  TenantRefAPITest.class,
+})
```

Comment:
Given that this suite has no shared logic, is the only purpose to enforce an execution order on these tests?

Reply:
The fixed order is only one of the reasons for using a test suite like this.

A purpose here is to use the Suites to bring in the tests rather than via an Inclusion in the pom.xml.
There is an AllTestSuite that is designed to pull in all the other test Suite classes and to control the order in which the individual test suites run.

I don't have any particular idea how to group them so I grouped them based on directory structure.
The "rest" tests are in the rest test suite.
The "api" tests are in the storage test suite.
The "service" tests are in the service test suite.

There is a bonus of being able to run the Suites as a set:
```
mvn test -Dtest=RestTestSuite
```

Reply:
> I don't have any particular idea how to group them so I grouped them based on directory structure.

That's likely because many of the tests are integration (in the infrastructure sense) tests and thus they aren't really organised in a particular way.